eng/Signing.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <ItemGroup>
+    <ItemsToSign Include="$(SparkPackagesDir)/**/*.nupkg" />
```

Comment:
Can we default these two `Dir` variables to a default location in the repo? I think there is a `PackagesDir` property defined in arcade for the shipping packages.

Reply:
> Can we default these two Dir variables to a default location in the repo? 

We could, but I think that would just make the build more complicated than it is now. 

Today, we:
1. build the nupkg and publish the worker in the first leg, then put them in an artifact.
2. On the second leg, we download the artifacts into the Staging directory. Sign everything and package the worker.
3. Publish the signed and packaged assets to the artifact.

If we put these two `Dir` variables into the repo, we would need to copy files around before and after (2) above.

> I think there is a PackagesDir property defined in arcade for the shipping packages.

Arcade has `$(ArtifactsPackagesDir)`

https://github.com/dotnet/arcade/blob/8052d0f693173d9bb99f6fa10611a6f4aafd4eb0/src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj#L17

But using that would mean the same copying files above, so I took the easier path of defining two new properties just for the signing operation. And the SparkWorkerPublishDir gets reused below when packaging the worker.